package.json

```diff
@@ -179,7 +194,8 @@
     "webpack": "4.41.5",
     "webpack-cli": "3.3.10",
     "webpack-fix-style-only-entries": "0.4.0",
-    "webpack-rtl-plugin": "2.0.0"
+    "webpack-rtl-plugin": "2.0.0",
+    "woocommerce": "git+https://github.com/woocommerce/woocommerce.git#try/wc-admin-e2e-test"
```

Comment:
`"woocommerce": "git+https://github.com/woocommerce/woocommerce.git#try/wc-admin-e2e-test"`

is `#try/wc-admin-e2e-test` a leftover or is it necessary to get the test utils?

Reply:
This is the main change we depend on in this branch: https://github.com/woocommerce/woocommerce/compare/try/add-wc-admin...try/wc-admin-e2e-test#diff-a39ea7ab3ac6262c137fbcb467aed169

Reply:
We can change this to `master` after https://github.com/woocommerce/woocommerce/pull/25636 lands.